homeassistant/components/netatmo/__init__.py

```diff
@@ -1,286 +1,102 @@
-"""Support for the Netatmo devices."""
-from datetime import timedelta
+"""The Netatmo integration."""
+
+# TODO:
+# * fix/add webhooks
```

Comment:
Use `hass_url = hass.helpers.network.async_get_external_url()` to get the url for webhooks. Method can return `None`.

Reply:
But can we move webhooks to a new PR, this one is getting big.

Reply:
Since webhooks are broken anyway I'll make this a separate PR.

Reply:
Can you remove the to do from the code.